README.md

```diff
@@ -32,67 +32,47 @@ var readStream = repo.archive.readFile('hello.txt', function (err, data) {
 var Dat = require('dat-js')
 
 var dat = new Dat()
-var repo = dat.create()
+var archive = dat.create()
 
 console.log('dat url is:', repo.url)
 
 // You can start reading/writing right away
-var writer = repo.archive.createWriteStream('hello.txt')
+var writer = archive.createWriteStream('hello.txt')
 
 writer.write('world')
-writer.end(function () { replicate(repo.url) })
+writer.end(function () { replicate(archive.url) })
 
 function replicate (url) {
   var clone = new Dat()
   var repo = clone.get(url)
 
-  var readStream = repo.archive.createReadStream('hello.txt')
+  var readStream = archive.createReadStream('hello.txt')
   readStream.on('data', function (data) {
     console.log(data.toString()) // prints 'world'
   })
 }
 ```
 
-#### Replication with a websocket gateway
-
-```js
-var Dat = require('dat-js')
-var concat = require('concat-stream')
-var pump = require('pump')
-
-var dat = new Dat({
-  gateway: 'ws://gateway.mauve.moe:3000'
-})
-var repo = dat.get('dat://SOME_ARCHIVE_URL')
-
-var readStream = repo.archive.createReadStream('hello.txt')
-
-pump(readStream, concat(function (data) {
-  console.log(data)
-}))
-```
-
 #### Persisting a created dat and loading it from storage
 
 ```js
 var Dat = require('dat-js')
-var db = require('random-access-idb')('dats')
 
 var dat = new Dat()
 
-var repo = dat.create({
-  db: db
+var archive = dat.create({
+  persist: true
 })
 
-repo.archive.writeFile('/example.txt', 'Hello World!', () => {
+archive.writeFile('/example.txt', 'Hello World!', () => {
   // Save it for later
-  localStorage.setItem('My_Repo', repo.url)
+  localStorage.setItem('My_Repo', archive.url)
```

Comment:
I guess this + the random-access-web options are confusing me. If it is persisting then why do you have to also manually persist it? We should specify *what* is being persisted.

Reply:
By default it's all in memory, I have the `persist` flag there for when you want to load archives into and from persistent storage. All the data in the archive would be getting persisted.

Reply:
But you still need to remember/save the archive key somewhere, right? That should be explicitly discussed.

Reply:
That's a good point. I'll add a comment to the create example and to the API docs

Reply:
That's a great point though. Why *should* they need to save the URL somewhere? It'd be nice if they could have named archives the way [multifeed](https://github.com/kappa-db/multifeed) does. 🤔 Probably for the next update, though.